INSTALL.md

```diff
@@ -19,7 +19,7 @@ Alexander Wenzel <Alexander.AW.Wenzel@bmw.de>
 * sudo apt install libqt5serialport5-dev
 * mkdir build
 * cd build
-* qmake ../BuildDltViewer.pro
+* cmake ..
```

Comment:
This complete INSTALL.md is always outdated like a newspaper, I would simply point to the CI build script and delete complete file